src/test/java/com/google/api/codegen/testdata/discoveries/java/java_cloudbilling.v1.json.discogapic.baseline

```diff
@@ -505,6 +514,10 @@ public final class ProjectBillingInfo implements Serializable {
     return projectId;
   }
 
+
+
+
+
```

Comment:
Done.
Added a new discovery doc billingtest.v1.json and the corresponding baseline files.